exercises/isogram/isogram_test.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'isogram'
+
+# Test data version:
+# 306975e
+class IsogramTest < Minitest::Test
+  def test_duplicates
+    str = 'duplicates'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_eleven
+    skip
+    str = 'eleven'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_subdermatoglyphic
+    skip
+    str = 'subdermatoglyphic'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_Alphabet
+    skip
+    str = 'Alphabet'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_thumbscrew_japingly
+    skip
+    str = 'thumbscrew-japingly'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_Hjelmqvist_Gryb_Zock_Pfund_Wax
+    skip
+    str = 'Hjelmqvist-Gryb-Zock-Pfund-Wax'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_Heizölrückstoßabdämpfung
```

Comment:
This code will filter the non-ASCII code points:

`str.unpack("U*").select{|c| c < 128 }.map(&:chr).join`